### DISCUSSION TITLE
Please Help: R-ESRGAN 4x+ and 4x+ Anime6B Error

### DISCUSSION THREAD

**Discussion by CrinstalMaiden** [lshqqytiger/stable-diffusion-webui-directml#419] [Q&A] [answered]
Hi everyone,

I'm new to SD and I'm currently using directml version to create AI pictures on my rx480. I have this annoying error which I can't upscale the generated picture with R-ESRGAN 4x+ and 4x+ Anime6B (ESRGAN_4x worked normally). Please help me since I'm not IT base and has zero knowledge on coding part of SD.

![Capture](https://github.com/lshqqytiger/stable-diffusion-webui-directml/assets/57482536/77389342-087e-49f3-8443-0a7a5c25aec4)

Many thanks,

**Comment by lshqqytiger** [comment]
#407 
(ZLUDA is not available on RX 480. Revert to 1.7 or use SD.Next)

**Reply by CrinstalMaiden** [reply]
So I'm using SD 1.8? Could you please show me how to revert back to 1.7 or SD.Next? I'm not very familiar with coding stuff so I'm very grateful

**Reply by lshqqytiger** [reply, marked as answer]
You are using SD-webui 1.8.
To revert to v1.7, run `git reset --hard 601f7e3704707d09ca88241e663a763a2493b11a` in cmd or powershell.

**Reply by CrinstalMaiden** [reply]
Oh lord. Thank you so much. It worked perfectly again
